core/src/main/scala/org/apache/spark/util/DependencyUtils.scala

```diff
@@ -59,8 +59,8 @@ private[spark] object DependencyUtils extends Logging {
    * @param uri Ivy URI need to be downloaded.
    * @return Tuple value of parameter `transitive` and `exclude` value.
    *
-   *         1. transitive: whether to download dependency jar of Ivy URI, default value is false
-   *            and this parameter value is case-sensitive. Invalid value will be treat as false.
+   *         1. transitive: whether to download dependency jar of Ivy URI, default value is true
+   *            and this parameter value is case-insensitive. Invalid value will be treat as false.
```

Comment:
Could you leave some comments like `this behaviour follows hive one...`?

Reply:
Fixed